src/components/summary/_macro.spec.js

```diff
@@ -307,8 +307,17 @@ describe('macro: summary', () => {
   describe('mode: general', () => {
     it('passes jest-axe checks', async () => {
       const $ = cheerio.load(renderComponent('summary', EXAMPLE_SUMMARY_BASIC));
+      const results = await axe($.html(), {
+        rules: {
+          dlitem: {
+            enabled: false,
+          },
+          'definition-list': {
+            enabled: false,
+          },
```

Comment:
I think these rules can be added using a variable because they are used so many times. Would also probably be good to add a comment as to why we are adding them

Reply:
Apart from this comment all looks good to me :)

Reply:
done